ccc-js/src/main/javascript/package-res/lib/tipsy.js

```diff
@@ -54,7 +54,16 @@
                 (instance.title || instance.text);
 
             // Allow deferred tooltip creation!
-            if(typeof title === 'function') title = title();
+            if(typeof title === 'function'){
```

Comment:
@bcostahitachivantara, imo, this kind of protection belongs directly to `jquery.tipsy.js`.
Luckily, the lib version is really "ours" (already largely customized by CCC). Would do it here: https://github.com/webdetails/ccc/blob/master/ccc-js/src/main/javascript/package-res/lib/jquery.tipsy.js#L231 .